### ISSUE TITLE
Footer view does not work properly

### ISSUE THREAD

**Issue by kjurkovic** [infinum/MjolnirRecyclerView#21]
Footer view is in xml defined with `match_parent` width. However when adding it to adapter it somehow gets to wrap_content and positions progress to left. 
this is footer view xml:
```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
              android:id="@+id/progress_container"
              android:layout_width="match_parent"
              android:layout_height="wrap_content"
              android:background="@color/default_background"
              android:orientation="vertical">

    <ProgressBar
        android:id="@+id/progress"
        style="?android:attr/android:progressBarStyleSmall"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="center_horizontal"
        android:layout_margin="@dimen/small_padding"/>
</LinearLayout>
```

this is how it is rendered: http://take.ms/HJpqX

Same view on ListView footer works properly

**Comment by zplesac** [comment]
Found the cause of this problem. 
Proposed solution is in PR #22 .

**Comment by zplesac** [comment]
Fixed and released in version 1.3.0.
